### PR TITLE
deployment: add post-processing

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Comment by matz-e** [comment]
Please retest

**Comment by matz-e** [comment]
Please retest

**Comment by matz-e** [comment]
Please retest

**Comment by matz-e** [comment]
Please retest

**Comment by matz-e** [comment]
Please retest

File: deploy/Jenkinsfile [hunk 1]
```diff
@@ -282,5 +286,35 @@ pipeline {
                 }
             }
         }
+
+        stage('Post-Mortem') {
+            parallel {
+                stage('Documentation') {
+                    steps {
+                        sh "ls"
```
Comment (pramodk) [inline deploy/Jenkinsfile:294]: if this you want to tackle later and if post-mortem stuff is ready, go ahead with merge

**Review by pramodk — APPROVED** [review]

**Pull request merged**